openwisp_controller/config/views.py

```diff
@@ -23,7 +23,12 @@ def get_default_templates(request, organization_id):
     backend = request.GET.get("backend", None)
     user = request.user
     authenticated = user.is_authenticated
-    if not authenticated and not user.is_staff:
+    if (
+        not authenticated
+        or not user.is_staff
+        or (not user.is_superuser and organization_id not in user.organizations_managed)
```

Comment:
I would use https://github.com/openwisp/openwisp-users/#is-manager-org since in this case should make it more readable